test-clients/src/main/java/com/hedera/services/bdd/suites/perf/crypto/CryptoTransferLoadTestWithInvalidAccounts.java

```diff
@@ -0,0 +1,89 @@
+package com.hedera.services.bdd.suites.perf.crypto;
+
+/*-
+ * ‌
+ * Hedera Services Test Clients
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hedera.services.bdd.spec.HapiApiSpec;
+import com.hedera.services.bdd.spec.HapiSpecOperation;
+import com.hedera.services.bdd.spec.utilops.LoadTest;
+import com.hedera.services.bdd.suites.perf.PerfTestLoadSettings;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+import java.util.function.Supplier;
+
+import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.cryptoTransfer;
+import static com.hedera.services.bdd.spec.transactions.crypto.HapiCryptoTransfer.tinyBarsFromTo;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.logIt;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.BUSY;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_ACCOUNT_ID;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.PLATFORM_TRANSACTION_NOT_CREATED;
+
+public final class CryptoTransferLoadTestWithInvalidAccounts extends LoadTest {
+	private static final Logger log = LogManager.getLogger(CryptoTransferLoadTestWithInvalidAccounts.class);
+
+	public static void main(String... args) {
+		parseArgs(args);
+
+		CryptoTransferLoadTestWithInvalidAccounts suite = new CryptoTransferLoadTestWithInvalidAccounts();
+		suite.runSuiteSync();
+	}
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(
+				runCryptoTransfers()
+		);
+	}
+
+	protected HapiApiSpec runCryptoTransfers() {
+		PerfTestLoadSettings settings = new PerfTestLoadSettings();
+
+		Supplier<HapiSpecOperation[]> transferBurst = () -> {
+			return new HapiSpecOperation[] { cryptoTransfer(
+					tinyBarsFromTo("0.0.1000000001", "0.0.1000000002", 1L))
+					.noLogging()
+					.signedBy(GENESIS)
+					.suppressStats(true)
+					.fee(100_000_000L)
+					.hasKnownStatusFrom(INVALID_ACCOUNT_ID)
+					.hasRetryPrecheckFrom(BUSY, PLATFORM_TRANSACTION_NOT_CREATED)
+					.deferStatusResolution()
```

Comment:
@qnswirlds  where will we be using this test ?

Reply:
Discussed privately. Will create a daily panel for this test.